controllers/users.js

```diff
@@ -63,11 +64,15 @@ const getUserById = async (req, res) => {
 
 const getUsers = async (req, res) => {
   try {
-    const allUsers = await userQuery.fetchUsers(req.query);
+    const { allUsers, next, prev } = await userQuery.fetchUsers(req.query);
 
     return res.json({
       message: "Users returned successfully!",
       users: allUsers,
+      links: {
+        next: next ? getPaginationLink(req.query, "next", next) : "",
```

Comment:
Change names from `next` and `prev` to `nextId` and `prevId`

Reply:
done